lib/taskana-cdi/pom.xml

```diff
@@ -35,18 +35,14 @@
   <dependencies>
     <dependency>
       <groupId>pro.taskana</groupId>
-      <artifactId>taskana-common-logging</artifactId>
```

Comment:
Why is taskana-common-logging deleted here?

Reply:
it's a transitive dependency from taskana-core. It is not needed it comes automatically.

But I gave you right it is more explicit when you include here. 

Just let me know if you think this way or the other is better. I have no real opinion about it :)

Reply:
@holgerhagen and I agree that it's better to remove the unneeded dependency